### DISCUSSION TITLE
It just refuses to work :(

### DISCUSSION THREAD

**Discussion by Yalcin20xd** [raiguard/ModernGadgets#287] [Q&A] [answered]
Hello!

Hwinfo64_v7.24-4770
Rainmeter v4.5.13
Modern Gadgets v1.8.2
Windows 11 Home 21H2

![1](https://user-images.githubusercontent.com/105592861/168474493-cdafbaf5-f7fe-46f6-9fdc-9fad12a7b21e.png)

Text files are like this.

![3](https://user-images.githubusercontent.com/105592861/168474490-4f521f94-cc44-475f-888a-126c61c2f80b.png)

Every single sensor has "_Enable reporting to Gadget_" and "_Report value in Gadget_" enabled

![4](https://user-images.githubusercontent.com/105592861/168474489-2640ea2f-4d30-47b6-9b62-e902fd49652b.png)

I don't think there is anything wrong with settings too.

![5](https://user-images.githubusercontent.com/105592861/168474487-2a977642-dc87-495a-a4bb-38b8540860d5.png)

and lastly Rainmeter has full control.

![2](https://user-images.githubusercontent.com/105592861/168474492-7108e07d-744d-4951-8ff1-acfb1ec6f791.png)

But.. I still get this. I cannot see where the problem is. 

Any help would be appreciated.

Edit: Also there is the Rainmeter log files.

![6](https://user-images.githubusercontent.com/105592861/168477435-1a71a658-b72e-4ea3-9142-a0ff6c41b974.png)

**Comment by SilverAzide** [comment, marked as answer]
It does not appear that you configured any of the variables in the HWiNFOSettings.inc file with the indexes from the list (as shown in your first screenshot).  For each of the variables in the HWiNFOSettings.inc file, you need to indicate which sensor index it uses.

For example:
```ini
HWiNFO-CPU0-Core0Clock=13
HWiNFO-CPU0-Core1Clock=14
HWiNFO-CPU0-Core2Clock=15
```
etc...  So you correctly configured HWiNFO, but you didn't configure the ModernGadgets skins.

**Reply by Yalcin20xd** [reply]
Huh. I feel silly now. It worked I can't believe I missed it. Thank you very much.
